package.json

```diff
@@ -31,5 +32,8 @@
     "./apps/*",
     "./packages/*"
   ],
-  "packageManager": "yarn@3.2.3"
+  "dependencies": {
```

Comment:
Should these be `dependencies` or `devDependencies`?

Reply:
I've removed it from this `package.json` as it's not needed here.